packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -176,7 +177,7 @@ export class GutenbergEditorPage extends BaseContainer {
 		await this.frame.click( selectors.publishPanelToggle );
 		await this.frame.waitForSelector( selectors.publishPanel );
 		await this.frame.click( selectors.publishButton );
-		await this.frame.waitForSelector( selectors.viewPostButton );
+		await this.frame.waitForSelector( selectors.publishSpinner, { state: 'hidden' } );
```

Comment:
This should not be necessary as it's not any different from waiting for the publish button to appear. It's also more prone to break the test, as the spinner element is more likely to change and/or appear after this line is called, which would immediately result in a false positive.

Reply:
Is the change from https://github.com/Automattic/wp-calypso/pull/53835/files#diff-8aeb2361aa173b5616012d0e178c97eee06325bc9dd1b7a9570037ada0238659R195 not sufficient?

Reply:
@WunderBart There has been occasional situations where the publish process fails due to timing, so the test step is left hanging. 

I can check if the change at Line 196 is sufficient, but part of the difficulty is we won't know until it starts running frequently on `trunk` for more data.

Reply:
Ran it a couple of times with this change and it seems to be doing good, no muted tests (other than HTTP304 issue I am working on)! 👍